ui/src/shared/presenters/index.js

```diff
@@ -110,11 +110,9 @@ function getRolesForUser(roles, user) {
 }
 
 export const buildDefaultYLabel = queryConfig => {
-  return queryConfig.rawText
-    ? ''
-    : `${queryConfig.measurement}.${_.get(
-        queryConfig,
-        ['fields', '0', 'field'],
-        ''
-      )}`
+  const {measurement, fields} = queryConfig
+  const fieldAlias = `${_.get(fields, ['0', 'alias'], '')}`
+  const field = `${_.get(fields, ['0', 'value'], '')}`
```

Comment:
Possibly, you could use `getFields`  from `shared/reducers/helpers/fields` and use the first element in the array to get alias and value.

This matters when you have a function of a function:

```sql
SELECT HOLT_WINTERS_WITH_FIT(FIRST("water_level"),10,4) FROM "NOAA_water_database"."autogen"."h2o_feet" WHERE "location"='santa_monica' AND time >= '2015-08-22 22:12:00' AND time <= '2015-08-28 03:00:00' GROUP BY time(379m,348m)
```


In your code it would incorrectly return empty string rather than water_level